### DISCUSSION TITLE
[A/L] (2021/22) Logica - Esercizio 3

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#329] [Esercizi A-L]
![image](https://user-images.githubusercontent.com/69389461/152324329-add80606-a171-46b0-a9f8-7017e36e5164.png)

**Comment by danieldinella** [comment]
Da A → B = 1 abbiamo o B = 1 o A e B = 0.
1. Nel primo caso abbiamo che B v C da come risultato 1 dunque a prescindere dal primo termine il risultato sarà 1. Nel secondo caso, essendo sia A che B = 0, possiamo semplificare la proposizione a C → C che darà 1 in qualsiasi caso. Dunque la proposizione è sempre vera.
2. Nel primo caso, avendo B = 1, riduciamo la proposizione a A ^ C  →  C , nel caso in cui C sia 1 abbiamo A → 1 = 1 mentre nel caso in cui C sia 0 avremo 0 → 0 = 1 . Nel secondo caso a prescindere da C, essendo sia A che B = 0, la proposizione si ridurrà a 0 → 0 = 1. Dunque la proposizione è sempre vera
3. Nel primo caso, avendo B = 1, se A = 1 otteniamo 0 ⟷ 1 che è falso, mentre nel caso in cui A = 0 abbiamo 1 ⟷ 1 che è vero. Nel secondo caso il risultato sarà 0 ⟷ 0 che è vera. Dunque la proposizione è falsa solo per A=1 e B=1.

**Comment by CiottoloMaggico** [comment]
![immagine](https://user-images.githubusercontent.com/79638284/217793533-8f6584eb-86ee-417f-a9e6-d8180bd8dec5.png)

**Comment by DavideGalilei** [comment]
> ⚠️ Questa soluzione interpreta il testo dell'esercizio come se le tre proposizioni siano congiunte, ovvero i casi dove siano vere le tre proposizioni contemporaneamente.

> ⚠️ Potrei aver interpretato male la consegna riguardando le soluzioni proposte sopra, ma terrò questo commento senza eliminarlo come reference.

Sapendo che $\large{A \to B}$ possiamo escludere una sezione dalla tavola della verità (evidenziata in rosso), e non considerarla.

- La prima espressione si può semplificare in $\large{\lnot{A} + B + C}$
- La seconda espressione si può semplificare in $\large{\lnot{A} + B + \lnot{C}}$
- La terza espressione si può semplificare in $\large{\lnot{A}}$

L'unica intersezione dove $\large{A \to B}$ e le tre formule sono vere è dove $\large{A = 0}$.

Quindi possiamo concludere con certezza che:
- $\large{A = 0}$
- la prima formula è sempre vera
- la seconda formula è sempre vera

![image](https://github.com/user-attachments/assets/458f718c-f8c4-4556-bd0f-4b6ca023468f)
